bot_commands.json

```diff
@@ -8,7 +8,11 @@
         "description": "create new queue"
     },
     {
-        "command": "/addmetoqueue",
-        "description": "Add yourself into the queue"
+      "command": "/addmetoqueue",
```

Comment:
Inconsistent formatting, use 4 space instead of 2

Reply:
You're welcome to fix this inconsistency in a separate pull request!